File: loan-repayment-java/src/test/java/com/loan/repayment/ApplicationIT.java
package com.loan.repayment;

import static org.junit.jupiter.api.Assertions.assertNotNull;

import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
import org.springframework.core.env.Environment;

import lombok.extern.slf4j.Slf4j;

/**
 * 
 * @author Ramachandra
 *
 */
@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
@Slf4j
class ApplicationIT {
    
    /**
     * Environment
     */
    @Autowired
    private Environment env;

    /**
     * This test for testing the Application main class for Code Coverage
     */
    @Test
    void main() {
        Application.main(new String[] {});
        assertNotNull(env, "Expected the environment as not null object");
    }

}

File: loan-repayment-kotlin/src/test/kotlin/com/loan/repayment/ApplicationIT.kt
package com.loan.repayment

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.boot.test.context.SpringBootTest.WebEnvironment
import org.springframework.core.env.Environment

/**
 *
 * @author Ramachandra
 */
@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
internal class ApplicationIT {
    /**
     * Environment
     */
    @Autowired
    private lateinit var env : Environment

    /**
     * This test for testing the Application main class for Code Coverage
     */
    @Test
    fun main() {
        assertNotNull(env , "Expected the environment as not null object")
    }
}